text/0000-enable-embroider.md

```diff
@@ -0,0 +1,190 @@
+---
+Stage: Accepted
+Start Date: 2021-04-27
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): Ember CLI
+RFC PR:
+---
+
+# Enable Embroider
+
+## Summary
+
+This RFC proposes enabling Embroider as the default build system for ember-cli.
+
+## Motivation
+
+The current build system provided by ember-cli (lib/ember-app.js) builds and combines your application and addon's code in a very dynamic manner. While this works, it comes with a cost of forcing additional complexities at build time while also bloating application sizes. Embroider moves away from this dyanmic behavior by upgrading your addons to be statically analysable into what is refered to as [V2 Packages](https://github.com/emberjs/rfcs/blob/master/text/0507-embroider-v2-package-format.md). Embroider can then operate on these packages to know exactly what is or is not needed apart of your application and can use that information to produce spec compliciant Javscript that all modern packagers (like Webpack) understand. At this point we can leverage the benifits of a particular packager such as Webpack's treeshaking abilities and dynamic imports (to name a few).
+
+Additional information about the motiviation and key features of Embroider can be [found here](https://github.com/embroider-build/embroider/blob/master/SPEC.md).
+
+## Detailed design
+
+This design will cover a few topics. The first discusses the explict file changes that happen for your application to use Embroider. Next it will cover how both new applications move to Embroider and how existing application will migrate to it. Lastly, it will cover some deprecated concepts that no longer apply under Embroider along with how we get community addon adoption.
```

Comment:
```suggestion
This design will cover a few topics. The first discusses the explicit file changes that happen for your application to use Embroider. Next it will cover how both new applications move to Embroider and how existing applications will migrate to it. Lastly, it will cover some deprecated concepts that no longer apply under Embroider along with how we get community addon adoption.
```